lib/Language/Souffle/Interpreted.hs

```diff
@@ -59,12 +63,19 @@ newtype SouffleM a
 --
 --   - __cfgDatalogDir__: The directory where the datalog file(s) are located.
 --   - __cfgSouffleBin__: The name of the souffle binary. Has to be available in
+--   - __cfgFactDir__: The directory where the initial input facts can be found
+--   if present. If Nothing, then a temporary directory will be used, during the
+--   souffle session.
+--   - __cfgOutputDir__: The directory where the output facts file(s) are created.
+--   If Nothing, it will be part of the template directory.
 --   \$PATH or an absolute path needs to be provided. Note: Passing in `Nothing`
 --   will fail to start up the interpreter in the `MonadSouffle.init` function.
```

Comment:
```suggestion
--   \$PATH or an absolute path needs to be provided. Note: Passing in `Nothing`
--   will fail to start up the interpreter in the `MonadSouffle.init` function.
--   - __cfgFactDir__: The directory where the initial input fact file(s) can be found
--   if present. If Nothing, then a temporary directory will be used, during the
--   souffle session.
--   - __cfgOutputDir__: The directory where the output facts file(s) are created.
--   If Nothing, it will be part of the temporary directory.
```